lib/pqa/import.rb

```diff
@@ -8,7 +8,7 @@ def initialize(pqa_service = PQA::ApiClient.from_settings)
 
     def run(date_from, date_to)
       query_api_and_update do
-        @pqa_service.questions(date_from, date_to, "Tabled")
+        @pqa_service.questions(Time.find_zone("UTC").parse(date_from.to_s), Time.find_zone("UTC").parse(date_to.to_s), "Tabled")
```

Comment:
@njpreddy Could you please try if it works?
```suggestion
@pqa_service.questions(date_from.utc, date_to.utc, "Tabled")
```